packages/framework/src/Migrations/AbstractMigration.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use PDO;
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainsConfigDefinition;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration as BaseAbstractMigration;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Yaml\Yaml;
+
+abstract class AbstractMigration extends BaseAbstractMigration implements ContainerAwareInterface
+{
+    /**
+     * @var \Symfony\Component\DependencyInjection\ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @param \Symfony\Component\DependencyInjection\ContainerInterface|null $container
+     */
+    public function setContainer(?ContainerInterface $container = null)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @return array
+     */
+    protected function getAllDomainIds(): array
+    {
+        return array_map(function ($domainConfig) {
+            return $domainConfig[DomainsConfigDefinition::CONFIG_ID];
+        }, $this->getParsedDomainConfigs());
+    }
+
+    /**
+     * @return array
+     */
+    protected function getCreatedDomainIds(): array
```

Comment:
I would name it something like `getDomainIdsFromSettingValues` to be more describing.